app/views/routes_into_teaching/steps/completed.html.erb

```diff
@@ -1,28 +1,105 @@
-        <%= render Content::ResultsBoxComponent.new(
+  <div class="row inset">
+    <div class="col col-845">
+      <h2 class="heading--box-blue">Your options</h2>
+      <p>Based on your answers, you may be eligible for the following teacher training options:</p>
+    </div>
+  </div>
+
+  <div class="row inset">
+    <div class="col col-845">
+      <% @results.each do |result| %>
+        <div class="col-space-s">
+          <%= render Content::ResultsBoxComponent.new(
           heading: result["title"],
           fee: result["course_fee"],
           course_length: result["course_length"],
           funding: result["funding"],
           text: result["description"],
-          link_text: "Find out more",
-          link_target: "/steps-to-become-a-teacher",
+          link_text: result["cta_text"],
+          link_target: result["cta_link"],
         ) %>
-      </div>
-    <% end %>
+        </div>
+      <% end %>
+    </div>
+  </div>
+
+  <div class="row inset">
+    <div class="col col-845">
+      <h2 class="heading--box-blue">Next steps</h2>
+
+      <%= render CallsToAction::SimpleComponent.new(icon: "icon-arrow-black") do %>
+        <h2 class="call-to-action__heading">Get personalised guidance</h2>
+        <p>Everything you need to know to start a career in teaching sent straight into your inbox. Tailored to your own situation, you'll get all the latest information as well as advice and support.</p>
+
+        <div class="row col-space-m">
+          <%= form_with builder: GOVUKDesignSystemFormBuilder::FormBuilder, url: mailing_list_step_path(:name), scope: :mailing_list_steps_name, method: :put do |f| %>
+            <%= f.govuk_text_field :first_name, autocomplete: "given-name" %>
+            <%= f.govuk_text_field :last_name, autocomplete: "family-name" %>
+            <%= f.govuk_email_field :email, autocomplete: "email" %>
+            <%= f.hidden_field :channel_id, value: params[:channel] || f.object&.channel_id.presence %>
+            <%= f.hidden_field :sub_channel_id, value: params[:sub_channel] || f.object&.sub_channel_id.presence %>
+            <%= f.hidden_field :accepted_policy_id, value: @privacy_policy.id %>
+            <div class="row col-space-s col-space-s-top">
+              <small>
+                Your details are protected under the terms of our <%= link_to("privacy notice (opens in new tab)", "/privacy-policy", { class: "link--black", target: :blank }) %>. This explains how we use your personal data. It's important you read it before signing up to receive emails.
+              </small>
+            </div>
+            <%= f.button "Next step", class: "button", data: { "prevent-double-click": true, "disable-with": "Next step" } %>
+          <% end %>
+        </div>
+      <% end %>
+    </div>
   </div>
 </section>
+
+<div class="row">
+  <div class="col-space-l-top">
+    <% if non_uk? %>
+      <%= render Content::QuoteWithImageComponent.new(
+        title: "Teaching as a non-uk...",
```

Comment:
This needs to have a heading level attributed to it @spencerldixon 

I _think_ it should be an h3 as it comes under the h2 of `Next steps`?

Reply:
However, looking in production, this component does not have a heading eg on https://getintoteaching.education.gov.uk/life-as-a-teacher/teaching-as-a-career/career-progression so I'm presuming we would need to look at this separately?

Reply:
I've changed QuoteWithImageComponent to render a h3 by default, but I've set this as an option, so we can always override this with another value if needed elsewhere